osbs/cli/main.py

```diff
@@ -352,7 +352,7 @@ def cmd_build(args, osbs):
         required_args = ['platform', 'release']
         unwanted_args = ['platforms']
         create_func = osbs.create_worker_build
-    elif args.orchestrator:
+    elif args.orchestrator and osbs.can_orchestrate():
```

Comment:
If you leave this line unchanged, the failure will be logged using the caught exception, something like:
```
Exception caught: OsbsValidationException("can't create orchestrate build when can_orchestrate isn't enabled")
```

Isn't that better? Otherwise the user will be tinkering around with their parameters trying to get their orchestrator build to work, and when they finally get something submitted it won't be the orchestrator build they asked for.